lib/devicetrust/testenv/fake_device_service.go

```diff
@@ -129,6 +140,65 @@ func (s *fakeDeviceService) EnrollDevice(stream devicepb.DeviceTrustService_Enro
 	return trace.Wrap(err)
 }
 
+func mustRandomBytes() []byte {
+	buf := make([]byte, 32)
+	_, err := rand.Read(buf)
+	if err != nil {
+		panic(err)
```

Comment:
Don't panic, let's handle this like all the other errors.

Reply:
0acf776324